### DISCUSSION TITLE
Where are the converted files?

### DISCUSSION THREAD

**Discussion by Anjetika** [audiamus/AaxAudioConverter#90] [Q&A] [answered]
It tells me that my aax files have been converted but they don't show up in file explorer. It said it should be in music/audiobook but no such file or folder was created.

**Comment by audiamus** [comment, marked as answer]
The converted files are will end up in the folder shown in the field next to the [Save to] button. When running the converter the first time, this field is empty and the converter will open a file dialog. If you accept the default then the destination folder will indeed become c:\users\<user>\music\audiobook.  

If that folder cannot be accessed or created for some reason, you should normally get an error message and the conversion should abort. With the [Save to] button to can select any other folder but you must have write access to it.
